web/src/frontend/fluent-core/components/DownloadManagerTask.ts

```diff
@@ -116,4 +117,24 @@ export class DownloadManagerTask extends FASTElement {
     private UpdateProgress = function (_: IDownloadTask, value?: number) {
         this.progress = value ?? 0;
     }.bind(this);
+
+    public ShowErrors() {
+        if(this.entry.Errors.length > 0) {
+            // TODO: Show all errors in a fancy error dialog ...
+            const message = this.entry.Errors.map(error => {
+                return `<div>${error.message}</div><pre>${error.stack}</pre>`;
+            }).join('<hr>');
+            window.open(null, '_blank', [
+                'titlebar=no',
+                'menubar=no',
+                'toolbar=no',
+                'ocation=no',

```

Comment:
Should be `location`

Reply:
Solved in [cceb962](https://github.com/manga-download/haruneko/pull/79/commits/cceb962e03416dbb02d502e29857cad8029fa64a)